pkgs/development/tools/build-managers/gup/.gitignore

```diff
@@ -0,0 +1 @@
+src.json.gup
```

Comment:
Not at this point - it's just:

```
#!bash -eu
update_script="$(which nix-prefetch-src)"
gup -u src.in.json "$update_script"
"$update_script" src.in.json "$1"
```

..which relies on [a script](https://github.com/timbertson/app-customisations/blob/master/bin/nix-prefetch-src) in https://github.com/timbertson/app-customisations/ being on `$PATH`. One day I might package it for easy public use, but I haven't put in that effort yet.